### PR TITLE
[MIG][11.0] website_sale_delivery: migration scripts

### PR DESCRIPTION
**Pull request by cubells**
Depends on:

- [x] delivery - #1103

cc @Tecnativa

### CODE REVIEW

File: addons/website_sale_delivery/migrations/11.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_moved_fields(
```
Comment (pedrobaeza) [inline addons/website_sale_delivery/migrations/11.0.1.0/post-migration.py]: Do this on pre

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
@pedrobaeza 

done!

**Review by pedrobaeza — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Rebased now that dependencies are merged.

**Comment by cubells** [comment]
@pedrobaeza done!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**